tests/jax/core_system_components/config_test.py

```diff
@@ -108,16 +196,16 @@ def test_add_multiple_configs(
 
 
 def test_add_config_twice(
```

Comment:
Is it not redundant to test the add  function multiple times?

Reply:
I don't think so, because we are trying to verify that the add method works as we expect when we use it in different settings.

Reply:
That sounds fair. You can't have too much testing :)